pkg/backend/apply.go

```diff
@@ -220,12 +220,14 @@ func PreviewThenPromptThenExecute(ctx context.Context, kind apitype.UpdateKind,
 			return changes, res
 		}
 
-		// If we had an original plan use it, else use the newly generated plan (might be nil if we've turned
+		// If we had an original plan use it, else if experimental use the newly generated plan (might be nil if we've turned
 		// plan generation off)
 		if originalPlan != nil {
 			op.Opts.Engine.Plan = originalPlan
-		} else {
+		} else if op.Opts.Engine.Experimental {
```

Comment:
Idea is this is the right place to change the prompt in a follow up PR such that if experimental we leave the prompt as is (yes/no) and use the plan, but if not experimental we change the prompt to (yes/no/with plan) and if with plan is select then use the plan, else fall into the nil case below.